src/neo/Ledger/StorageItem.cs

```diff
@@ -1,15 +1,48 @@
 using Neo.IO;
+using Neo.SmartContract;
 using System.IO;
 
 namespace Neo.Ledger
 {
     public class StorageItem : ICloneable<StorageItem>, ISerializable
     {
-        public byte[] Value;
+        private byte[] value;
+        private IInteroperable interoperable;
         public bool IsConstant;
 
         public int Size => Value.GetVarSize() + sizeof(bool);
 
+        public byte[] Value
+        {
+            get
+            {
+                if (value is null && interoperable != null)
+                    value = BinarySerializer.Serialize(interoperable.ToStackItem(null), 4096);
```

Comment:
I think that `ISerializable` should be better, maybe we want to hide some values to the SmartContract, but we want to store more information.
Also I think that we should remove `value` variable. If we query this Property, and make some changes in the `interoperable` this changes won't be applied to the next query of `Value`

Reply:
So you should call `GetInteroperable()` every time you make changes to the `interoperable`.

Reply:
And what do you think about this?

> I think that ISerializable should be better, maybe we want to hide some values to the SmartContract, but we want to store more information.

Reply:
How?

Reply:
I think that instead of `IInteroperable` we could use `ISerializable`

Reply:
`IInteroperable` is used to allow these storages to be easily read by smart contracts.